monai/data/sliding_window_inference.py

```diff
@@ -102,7 +113,7 @@ def sliding_window_inference(inputs, roi_size, sw_batch_size, predictor):
     return output_image[..., : original_image_size[0], : original_image_size[1]]  # 2D
 
 
-def _get_scan_interval(image_size, roi_size, num_spatial_dims):
+def _get_scan_interval(image_size, roi_size, num_spatial_dims, overlap):
     assert len(image_size) == num_spatial_dims, "image coord different from spatial dims."
     assert len(roi_size) == num_spatial_dims, "roi coord different from spatial dims."
 
```

Comment:
Do we still need to initialize `scan_interval ` with all `1`?

Reply:
Can initialize with None also. this 1 is meaningless.

Reply:
I think we can simplify to:
```py
    scan_interval = list()
    for i in range(num_spatial_dims):
        if roi_size[i] == image_size[i]:
            scan_interval.append(int(roi_size[i]))
        else:
            scan_interval.append(int(roi_size[i] * (1 - overlap)))
```
What do you think?
Thanks.

Reply:
Changed.

Pre-initialization is significantly faster, but it only matters for large arrays.